lib/java/com/google/android/material/internal/CollapsingTextHelper.java

```diff
@@ -571,34 +628,62 @@ public void draw(Canvas canvas) {
       final boolean drawTexture = useTexture && expandedTitleTexture != null;
```

Comment:
If we care enough to check that expandedTitleTexture exists, maybe we should also check that the other textures exist?

Reply:
Yes, that makes sense. Fixed in 5c073ae. 🙂